Cargo.toml

```
@@ -40,4 +40,4 @@ zeroize = "1.4"
 [dev-dependencies]
 blake2 = "0.9"
 multibase = "0.9"
-serde_json = "1.0"
+bincode = "1.3"
```

```suggestion
bincode = "1"
```
Because major version is >0 it should be ok to omit the minor version.

---

All the other dependencies (e.g. `zeroize`) still use `x.y` pattern so I'm thinking it's better to be consistent?

---

Sure.  In a library crate I feel like we should switch all of them to use `x` instead of `x.y` where possible but it's not that big a deal to me.  I thought I remember seeing similar examples in RustCrypto cargo files but can't seem to find any examples at the moment.